### DISCUSSION TITLE
Certificate Expired

### DISCUSSION THREAD

**Discussion by ailox** [uxbox/uxbox#796] [General]
I don't know if this is the right place for this, but the https://penpot.app/ Certificate has expired. Since HSTS is enabled (which is very nice!), nobody is able to use the Project currently.

![image](https://user-images.githubusercontent.com/33927916/112717657-6ef76600-8eee-11eb-968a-64beb464abf3.png)
![image](https://user-images.githubusercontent.com/33927916/112717648-6141e080-8eee-11eb-8dc2-46088a1dfa4f.png)

**Comment by ailox** [comment]
It was resolved.

**Comment by Alotor** [comment]
Sorry for the inconvenience. Was very unlucky for this to happen on a weeked :disappointed:
